components/automate-ui/src/app/pages/data-feed-create/data-feed-create.component.html

```diff
@@ -138,15 +138,15 @@ <h3 class="bolder-font">{{item.name}}</h3>
                   data-cy="add-token-type"
                   id="token-type-input"
                   autocomplete="off"
-                  [readonly]="tokenToggle"/>
-                <chef-button secondary data-cy="toggle-type" (click)="toggleTokenType()">
+                  [readonly]="tokenToggle && !showFields.customToken"/>
+                <chef-button secondary data-cy="toggle-type" (click)="toggleTokenType()" *ngIf="!showFields.customToken">
                   Edit
                 </chef-button>
               </div>
             </label>
```

Comment:
```html
<chef-error
  *ngIf="createForm.get('tokenType').hasError('required') && createForm.get('tokenType').dirty">
  token type is required.
</chef-error>
```
Please add this after `</label>` as it will be required when token type is empty